infrastructure/main.tf

```diff
@@ -40,6 +40,9 @@ module "submit-your-appeal-frontend" {
     EXTERNAL_HOSTNAME            = "${var.env != "preview" ? var.sya_hostname : "${var.deployment_namespace}-sscs-tribunals-frontend-${var.env}.service.${local.aseName}.internal"}"
     HPKP_SHA256                  = "${data.vault_generic_secret.hpkp_sya_sha_1.data["value"]}"
     HPKP_SHA256_BACKUP           = "${data.vault_generic_secret.hpkp_sya_sha_2.data["value"]}"
+    EVIDENCE_UPLOAD_ENABLED      = ""
+    UPLOAD_EVIDENCE_URL          = "${local.ApiUrl}/evidence/upload"
+    LOG_LEVEL                    = "debug"    
```

Comment:
This is the kind of things I am concerned about. Infrastructure changes done in the main configuration that we don't want to bring in yet.

Reply:
We can very easily remove this. I shall remove.